types/params.go

```diff
@@ -116,6 +125,15 @@ func DefaultVersionParams() VersionParams {
 	}
 }
 
+func DefaultSynchronyParams() SynchronyParams {
+	// TODO(@wbanfield): Determine experimental values for these defaults
+	// https://github.com/tendermint/tendermint/issues/7202
```

Comment:
Also has the TODO here been completed? Would be good to have a short description of how these values were derived